var/spack/repos/builtin/packages/gdrcopy/package.py

```diff
@@ -24,13 +24,19 @@ class Gdrcopy(MakefilePackage):
     # Don't call ldconfig: https://github.com/NVIDIA/gdrcopy/pull/229
     patch("ldconfig.patch", when="@2.0:")
 
+    depends_on("check")
+    depends_on("cuda")
```

Comment:
If a package inherits from `CudaPackage`, it automatically:
```
depends_on("cuda", when="+cuda")
```
so this is not needed. If you need to always depend on `cuda` you can either:
```
require("+cuda")
```
or
```
conflicts("~cuda")
```
when inheriting from a `CudaPackage`

Reply:
Thanks!  I did not know about  `require`.  I've added that now.